calibrators/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/calibrators/ceres_calibrator.py

```diff
@@ -18,19 +18,52 @@
 import threading
 from typing import Dict
 from typing import List
+from typing import Tuple
 
 from intrinsic_camera_calibrator.board_detections.board_detection import BoardDetection
 from intrinsic_camera_calibrator.calibrators.calibrator import Calibrator
-from intrinsic_camera_calibrator.camera_model import CameraModel
+from intrinsic_camera_calibrator.camera_models.camera_model import CameraModelEnum
+from intrinsic_camera_calibrator.camera_models.camera_model_factory import make_camera_model
+from intrinsic_camera_calibrator.camera_models.ceres_camera_model import CeresCameraModel
 from intrinsic_camera_calibrator.parameter import Parameter
 
 
 class CeresCalibrator(Calibrator):
     def __init__(self, lock: threading.RLock, cfg: Dict = {}):
         super().__init__(lock, cfg)
+        self.radial_distortion_coefficients = Parameter(int, value=3, min_value=0, max_value=3)
+        self.rational_distortion_coefficients = Parameter(int, value=3, min_value=0, max_value=3)
+        self.use_tangential_distortion = Parameter(
+            bool, value=True, min_value=False, max_value=True
+        )
+        self.pre_calibration_num_samples = Parameter(int, value=6, min_value=1, max_value=100)
+        self.regularization_weight = Parameter(float, value=0.001, min_value=0.0, max_value=1.0)
```

Comment:
Leaving a placeholder comment to remind ourselves to leave the default values and the max value after seeing the experiments related to this

Reply:
Marking this as resolved since this value seemed like a very good one (internal evidence)